Installing-and-Configuring/index.md

```diff
@@ -4,4 +4,78 @@ title: Installing and configuring
 permalink: /Install-and-Configure
 nav_order: 4
 has_children: yes
+has_toc: false
 ---
+
+Installing and configuring Heads
+===
+
+Required equipment
+---
+
+To install Heads on a physical device, you will need:
+
+* Supported motherboard or laptop ([see below](#supported-devices))
+* A OTP security key ([see below](#security keys))
```

Comment:
Language that was agreed upon is USB Security Dongle (OTP) and HOTP enabled USB Security Dongle (For Nitrokey Pro/Nitrokey Storage and Librem Key) for which boards need to have [HOTP_KEY added in board config]( https://github.com/osresearch/heads/blob/master/boards/x230-hotp-verification/x230-hotp-verification.config#L28)

Reply:
@tlaurion Why do only certain broads have HOTP_KEY enabled?

Reply:
I added a note about this HOTP device support next to the USB Security Dongle list.